src/TreeHouse/KeystoneBundle/Resources/config/security.yml

```diff
@@ -1,46 +1,44 @@
-parameters:
-  tree_house.keystone.user_provider.default.class: TreeHouse\KeystoneBundle\Security\User\UserProvider
-  tree_house.keystone.token_authenticator.class: TreeHouse\KeystoneBundle\Security\Authentication\TokenAuthenticator
-  tree_house.keystone.authentication.listener.http_post.class: TreeHouse\KeystoneBundle\Security\Firewall\HttpPostAuthenticationListener
-  tree_house.keystone.authentication.provider.user.class: Symfony\Component\Security\Core\Authentication\Provider\DaoAuthenticationProvider
-  tree_house.keystone.encoder.token.class: TreeHouse\KeystoneBundle\Security\Encoder\TokenEncoder
-  tree_house.keystone.http.entry_point.class: TreeHouse\KeystoneBundle\Security\Http\KeystoneEntryPoint
-
 services:
   tree_house.keystone.user_provider.default:
-    class: %tree_house.keystone.user_provider.default.class%
+    public: true
+    class: TreeHouse\KeystoneBundle\Security\User\UserProvider
     arguments:
-      - @doctrine
-      - %tree_house.keystone.model.user.class%
+      - '@doctrine'
+      - '%tree_house.keystone.model.user.class%'
 
   tree_house.keystone.token_authenticator:
-    class: %tree_house.keystone.token_authenticator.class%
+    public: false
+    class: TreeHouse\KeystoneBundle\Security\Authentication\TokenAuthenticator
     arguments:
-      - @tree_house.keystone.token_manager
-      - @security.user_checker
+      - '@tree_house.keystone.token_manager'
+      - '@security.user_checker'
 
   tree_house.keystone.authentication.listener.http_post:
-    class: %tree_house.keystone.authentication.listener.http_post.class%
+    public: false
+    class: TreeHouse\KeystoneBundle\Security\Firewall\HttpPostAuthenticationListener
     arguments:
-      - @security.context
-      - @security.authentication.manager
+      - '@security.token_storage'
+      - '@security.authentication.manager'
       - null # provider-shared key
-      - @logger
+      - '@logger'
     tags:
       - { name: monolog.logger, channel: security }
 
   tree_house.keystone.authentication.provider.http_post:
-    class: %tree_house.keystone.authentication.provider.user.class%
+    public: false
+    class: Symfony\Component\Security\Core\Authentication\Provider\DaoAuthenticationProvider
     arguments:
-      - @tree_house.keystone.user_provider
-      - @security.user_checker
+      - '@tree_house.keystone.user_provider'
+      - '@security.user_checker'
       - null # provider-shared key
-      - @security.encoder_factory
+      - '@security.encoder_factory'
 
   tree_house.keystone.encoder.token:
-    class: %tree_house.keystone.encoder.token.class%
+    public: false
+    class: TreeHouse\KeystoneBundle\Security\Encoder\TokenEncoder
     arguments:
-      - %secret%
+      - '%secret%'
 
   tree_house.keystone.http.entry_point:
-    class: %tree_house.keystone.http.entry_point.class%
+    public: false
+    class: TreeHouse\KeystoneBundle\Security\Http\KeystoneEntryPoint
```

Comment:
I saw you wrapped **all** values in quotes in other YAML files, why not here (for the classnames at least?)